plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -118,6 +119,7 @@ data DefaultUni a where
     DefaultUniBLS12_381_G1_Element :: DefaultUni (Esc BLS12_381.G1.Element)
     DefaultUniBLS12_381_G2_Element :: DefaultUni (Esc BLS12_381.G2.Element)
     DefaultUniBLS12_381_MlResult :: DefaultUni (Esc BLS12_381.Pairing.MlResult)
+    DefaultUniValue :: DefaultUni (Esc Value)
```

Comment:
I thought we agreed to call it `MaryValue`? Let's not repeat ledger folks' mistakes
<img width="920" height="165" alt="image" src="https://github.com/user-attachments/assets/253d90cb-d919-4d37-b4c6-e3c0790a7c80" />

Reply:
I don't recall the discussion - do you have a pointer? It's always called Value in plutus, and I'm hesitant to make a name 80% longer unless there's a clear advantage.

Reply:
But _only_ in the CIP name - not in the proposed types and signatures. There's currently no plan to introduce a new kind of Value, and the name is changeable later, so I see no point of speculatively doing so.